src/pyggdrasil/visualize/_tree.py

```diff
@@ -172,3 +169,52 @@ def plot(
     plt.savefig(fullpath + ".svg", bbox_inches="tight")
     plt.close()
     logger.info(f"Saved tree plot to {fullpath}.svg")
+
+
```

Comment:
Below: specialized plotting functions, that define arguments for the most general plotting function automatically. `plot_tree`